jsconfig.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
What is the purpose of this file?

Reply:
Ah, it's a configuration file for Visual Studio Code (https://code.visualstudio.com/docs/languages/jsconfig)